### ISSUE TITLE
How to set different weight decay values for different modules of the model

### ISSUE THREAD

**Issue by AlfredQin** [Deci-AI/super-gradients#2018]
### 💡 Your Question

Hi, I am trying to let the parts of my model have different weight decay value during training. For example, the backbone has weight decay 0.05 while the head has 0.01.  But I don't know how to implement it in the yaml configuration. Currently I can set different learning rate for each part of the model like this: 
training_hyperparams:
  initial_lr:
    backbone: 1e-5
    head: 1e-4
    default: 1e-4
  Howerver I can't find available method to set weight decay like what I do on learning rate configuration. Could you please help me fix it ? Thank you !
